modules/engine/src/index.ts

```diff
@@ -67,6 +69,8 @@ export class VectorEngine implements IVectorEngine {
 
   private readonly restoreLocks: { [channelAddress: string]: string } = {};
 
+  private auctionResponses: Array<NodeResponses.RunAuction> = [];
```

Comment:
global store for auction responses, maybe it should be a variable inside scope of runAuction method only